js/components/controlBar.js

```diff
@@ -112,13 +113,28 @@ var ControlBar = React.createClass({
     } else {
       this.toggleQualityPopover();
       this.closeCaptionPopover();
+      this.closeConfigPopover();
     }
   },
 
-  toggleQualityPopover: function() {
```

Comment:
What's the reason to rename this method? It calls `toggleVideoQualityPopOver` anyway.